src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -38,25 +40,30 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
-                        PREFIX_ADDRESS, PREFIX_BIRTHDAY_DATE, PREFIX_INSTRUMENT, PREFIX_TAG);
+                        PREFIX_ADDRESS, PREFIX_BIRTHDAY_DATE, PREFIX_MATRICULATIONYEAR, PREFIX_BIRTHDAY_DATE,
```

Comment:
There is a duplicate `PREFIX_BIRTHDAY_DATE`. You might need to change one of them to `PREFIX_INSTRUMENT` as it doesn't account for the instrument field currently.

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.